web/client/components/map/openlayers/DrawSupport.jsx

```diff
@@ -124,7 +125,7 @@ export default class DrawSupport extends React.Component {
         if (!newProps.drawStatus && this.selectInteraction) {
             this.selectInteraction.getFeatures().clear();
         }
-        if ( this.props.drawStatus !== newProps.drawStatus || this.props.drawMethod !== newProps.drawMethod || this.props.features !== newProps.features) {
+        if ( this.props.drawStatus !== newProps.drawStatus || this.props.drawMethod !== newProps.drawMethod || this.props.features !== newProps.features || this.props.drawStatus === "simpleDrag" || newProps.drawStatus === "simpleDrag") {
```

Comment:
With this solution the addSimpleDragOperation is activated on every update of props (eg map move, add new location, ..) and each new translate interaction is added to ol and never removed

![image](https://user-images.githubusercontent.com/19175505/93999546-3d161300-fd96-11ea-914e-f66978f71d18.png)

Following the approach of this component you should have a start and a end status (eg. startSmpleDrag, endSimpleDrag) and the end should remove all the interaction. The component that use the DragSupport should start and end it one per session without continuosly mount and unmount the component on each prop update.

@offtherailz please could you take a look at this including the part on the enhancers where the DrawSupport is used?

Reply:
With the experience for support found in Swipe tool, maybe @AGMETEOR can find a solution to this. Please let me know if you still need my opinion here, when you start again working on it.

Reply:
@offtherailz , should I create a new support maybe called "SimpleDrag"? If so, it should only be for OL implementation, right?

Reply:
I Don't know, It depends how much time it takes and if this solves this problem. Anyway the crucial point is to solve the issue notified by @allyoucanmap, that knows how Swipe integration is done too, so I think you can solve this problem together. 
I only said to ask me if you need again my help.